tests/imodels-access-backend-tests/src/BackendiModelsAccess.test.ts

```diff
@@ -188,6 +188,19 @@ describe("BackendIModelsAccess", () => {
   });
 
   describe("CreateNewIModel", () => {
+    it("should create new empty iModel", async () => {
+      await IModelHost.startup();
```

Comment:
This line seems potentially problematic. IModelHost should be started / shut down once per test suite run. We could end up with hanging tests potenitally.

Reply:
added `IModelHost.startup` call to `before` hook, `IModelHost.shutdown` to `after` hook